apps/web/components/apps/App.tsx

```diff
@@ -43,8 +43,10 @@ const Component = ({
   const router = useRouter();
 
   const mutation = useAddAppMutation(null, {
-    onSuccess: () => {
-      showToast(t("app_successfully_installed"), "success");
+    onSuccess: (data: any) => {
+      if (data.showToast !== undefined && data.showToast) {
```

Comment:
This is to fix the false positive "App successfully installed" toast message
cc: @hariombalhara